openfisca_core/scripts/run_test.py

```diff
@@ -57,8 +57,8 @@ def main():
         'verbose': args.verbose,
         'force': args.force,
         'name_filter': args.name_filter,
-        'default_relative_error_margin': args.relative_error_margin,
```

Comment:
I favor explicitness over shortness. Plus, I doubt the usefulness and usage frequency of this specific option, so I wouldn't care about length.